src/renderer/shared/api/xcm/lib/types.ts

```diff
@@ -43,6 +44,21 @@ export type NetworkBaseWeight = {
   [chainId: string]: string;
 };
 
+export type DeliveryFeeConfig = {
+  type: 'exponential';
+  factorPallet: 'ParachainSystem' | 'XcmpQueue' | 'Dmp';
+  sizeBase: string;
+  sizeFactor: string;
+  alwaysHoldingPays: boolean;
+};
+
+export type NetworkDeliveryFee = {
+  [chainId: string]: {
```

Comment:
why not `ChainId`?

Reply:
Because in config chainid is in string format, not hex string